sdk/react/bottomOverlay.js

```diff
@@ -100,6 +100,12 @@ var BottomOverlay = React.createClass({
     }
   },
 
+  componentWillReceiveProps: function(nextProps) {
+    if (Math.abs(this.props.playhead - nextProps.playhead) >= 0.2) {
```

Comment:
can you add a comment here?  is this just an assumption that we send updates that are smaller than .2?  maybe instead you can do this.props.playhead != nextProps.playhead?